sonar-html-plugin/src/test/java/org/sonar/plugins/html/core/HtmlSensorTest.java

```diff
@@ -72,7 +72,8 @@ public class HtmlSensorTest {
 
   @Before
   public void setUp() {
-    HtmlRulesDefinition rulesDefinition = new HtmlRulesDefinition();
+    final SonarRuntime sonarRuntime = SonarRuntimeImpl.forSonarQube(Version.create(8, 9), SonarQubeSide.SCANNER, SonarEdition.COMMUNITY);
```

Comment:
_curiosity_ It is not the first time I have seen a hardcoded version in tests. Why shouldn't we put this in a global variable or a config file?

Reply:
No reason :D
I can indeed put it in a constant somewhere. I just updated the test one after the other and did not think of it :+1:

Reply:
Alright !